train.rb

```diff
@@ -52,6 +49,10 @@ def print_stations
     puts "\n#{@route.get_stations(@current_station)}\n"
   end
 
+  def to_s
+    "Number #{@number}, type #{@type}, carriages: #{@carriages.map(&:to_s)}"
+  end
+
   private
 
   def transfer(start, finish)
```

Comment:
add rubocop and it should pass all tests. Only after that, you can push your changes!